src/libraries/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj

```diff
@@ -21,6 +21,9 @@
     <Compile Include="$(CommonPath)Interop\Unix\Interop.Errors.cs"
              Link="Common\Interop\Unix\Interop.Errors.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsAnyOS)' == 'true'">
+    <Compile Include="System\ComponentModel\Win32Exception.Default.cs" />
+  </ItemGroup>
```

Comment:
We are using `Interop.Errors.cs` for Browser in other libraries. Why we are not doing the same here?

Reply:
We wanted to have a platform independent implementation here, which could be used for any other future platforms(similar to browser)

Reply:
The primary goal should be for the platform assemblies to be self-consistent.

Everything else in the platform that deals with error codes delegates to `Interop.Errors.cs` on both Unix and Browser. Changing this assembly to be platform neutral is introducing inconsistency. It is not a huge deal for this one - just formatting of the error messages - but I believe it is going in a wrong direction.

Reply:
Actually, if we are OK using Interop.Error.cs and we think that will work we can undo this and create a browser config.  Originally I suggested this route because I thought someone said `Interop.Sys.StrError` was not going to work on browser.

Reply:
Yep, @marek-safar said that here: https://github.com/dotnet/runtime/pull/37944#discussion_r442440300

Reply:
I just tried `Interop.Sys.StrError` and it worked, so we should be fine using it.